app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -70,14 +72,22 @@ func (s *URLMapSyncer) EnsureURLMap(lbName string, ing *v1beta1.Ingress, beMap b
 	if err == nil {
 		fmt.Println("url map", name, "exists already. Checking if it matches our desired url map", name)
 		glog.V(5).Infof("Existing url map: %v\n, desired url map: %v", existingUM, desiredUM)
+		// Fingerprint is required (and we get an error if it's not set).
+		// TODO(G-Harmon): Figure out how to properly calculate the fp. Using Sha256 returned a googleapi error.
```

Comment:
Also mention why using the existing fingerprint is wrong?

Reply:
sure, done.